### DISCUSSION TITLE
Where can I find the settings again?

### DISCUSSION THREAD

**Discussion by mbakker7** [pastas/pastas#773] [General]
`pastas` has a bunch of default settings for certain stresses, but I always forget where I can find those. Can you point me to these settings again?

**Comment by dbrakenhoff** [comment]
`ps.rcParams["timeseries"]`

**Comment by raoulcollenteur** [comment]
Some more explanation can also be found here: https://pastas.readthedocs.io/latest/userguide/time_series.html
